src/containers/shared/utils.js

```diff
@@ -78,6 +78,54 @@ const FORMAT_PRICE_DEFAULT_OPTIONS = {
   padding: 0,
 }
 
+export const isLaterVersion = (source, target) => {
+  if (source === ' N/A ') return false
+  if (target === ' N/A ') return true
+  const sourceDecomp = source.split('.')
+  const targetDecomp = target.split('.')
+  // Compare major version
+  if (parseInt(sourceDecomp[0], 10) > parseInt(targetDecomp[0], 10)) return true
+  // Compare minor version
+  if (
+    parseInt(sourceDecomp[0], 10) === parseInt(targetDecomp[0], 10) &&
```

Comment:
Instead of using `parseInt` over and over again you can extract these into variables :) 

```
const left = parseInt(sourceDecomp[0], 10)
const right = parseInt(sourceDecomp[1], 10)
```

General rule: If you are doing the same computation over and over again, can you calculate once and reuse throughout?